orttraining/orttraining/core/graph/adasum_optimizer_graph_builder.cc

```diff
@@ -93,19 +68,87 @@ Status AdasumOptimizerGraphBuilder::BuildOptimizerNode(
       opt_configs, graph_defs,
       new_initializers,
       output_weight_argdefs, output_gradient_argdefs,
-      // Always enable grad clipping for Adasum
-      true /*enable_grad_clipping*/));
+      opt_graph_config_.enable_grad_norm_clip));
 
   return Status::OK();
 }
 
+static Status AddNcclAllReduceForGradientsWithGroups(
+    std::vector<ArgDef>& gradient_argdefs,
+    ArgDef& fused_gradient_argdef,
+    GraphAugmenter::GraphDefs& graph_defs,
+    ArgDef& fused_allreduce_output,
+    const int64_t group_type = 0) {
```

Comment:
Could this be one of the enum values / use an enum class for readability, or is this something different?

Reply:
ah Yes, I left it out by mistake. will change it to the enum.